### PR TITLE
Small improvements for before 1.0

### PR DESCRIPTION
**Pull request by ZeWaka**
rust-g compile time without caching: 11m20s linux, 13m41s windows
with cache: 4m8s linux, 4m34s windows

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ open("rust_g", O_RDONLY|O_NONBLOCK|O_LARGEFILE|O_DIRECTORY|O_CLOEXEC) = -1 ENOTD
 If you're still having problems, ask in the [Coderbus Discord]'s
 `#tooling-questions` channel.
 
-You can also try [tgstation]'s IRC, `#coderbus` on Rizon, but it is usually
+You can also try [/tg/station]'s IRC, `#coderbus` on Rizon, but it is usually
```
Comment (AffectedArc07) [inline README.md:181]: Isn't the actual service dead?

Comment (ZeWaka) [reply README.md:181]: no

Comment (ZeWaka) [reply README.md:181]: image

Comment (ZeWaka) [reply README.md:181]: it is just bouncer people though

Comment (AffectedArc07) [reply README.md:181]: Must have been down for maintenance last time I checked it then.

Comment (JohnFulpWillard) [reply README.md]: https://github.com/tgstation/tgstation/pull/63207

**Pull request merged**